src/sass/components/_team-members.scss

```diff
@@ -0,0 +1,23 @@
+// Team Member
+// ===========
+
+// Add/Edit Team Members Modal
+// ---------------------------
+.team-member-grid {
+  > div {
+    align-items: center;
+    display: flex;
+  }
+}
+
+.team-member-username {
+  margin-left: $spacing-x-small;
+}
+
+// Assigned Team Member Cards
+// --------------------------
+.card-in-list {
+  h2 {
```

Comment:
Can't edit markup so my next best scenario was to use my own class to target the card heading here

Reply:
That makes sense, but the `card-in-list` name doesn't feel very specific to this situation? I think this is all it does?